pkg/scan/filter.go

```diff
@@ -12,9 +12,10 @@ import (
 const (
 	AdvisoriesSetResolved = "resolved"
 	AdvisoriesSetAll      = "all"
+	AdvisoriesSetPending  = "pending"
```

Comment:
We should account for this new input option in the command help, too.

E.g.: https://github.com/wolfi-dev/wolfictl/blob/main/pkg/cli/scan.go#L77-L81

Reply:
Updated the command and docs.